tools/protoxform/protoprint.py

```diff
@@ -553,12 +568,79 @@ def format_reserved(enum_or_msg_proto):
     return reserved_fields
 
 
+def validate_deprecation_version(version: str):
+    """Validates that a given version string is of format X.Y, where X and Y are
+  integers, X>0, and Y>=0.
+
+  Args:
+    version: a minor version deprecation annotation value (see api/envoy/annotations/deprecation.proto)
+
+  Returns:
+    True iff the given string represents a valid X.Y version.
+  """
+    if version.count('.') != 1:
+        return False
+    # Validate major and minor parts.
+    try:
+        major, minor = [int(x) for x in version.split('.')]
+        if major <= 0:
+            return False
+        if minor < 0:
+            return False
+    except ValueError:
+        return False
+    return True
```

Comment:
Should this belong in `utils`? Good candidate for unit tests.

Reply:
Moved, and added tests.